services/cypress/README.md

```diff
@@ -0,0 +1,284 @@
+# Cypress
+This service deploys our Cypress app for e2e testing locally and CI. An important note is that the actual Cypress package is installed in the root `node_modules` this was the only way we could get Cypress commands to work in CI. The reason is still unknown, but we suspect it because some Cypress commands need a yarn.lock in the same directory where the package is installed and in our monorepo we only have a yarn.lock at the root directory.
+
+### Deploying locally
+  - Run local Cypress app and test against the local app deployment.
+    - `./dev test browser`
+  - We can also run the local Cypress app against a review app deployment.
+    - ```
+      REACT_APP_AUTH_MODE='AWS_COGNITO' TEST_USERS_PASS='<Test user pass here>' API_URL='<api url here>' COGNITO_USER_POOL_ID='<user pool id here>' COGNITO_REGION='<region here>' COGNITO_IDENTITY_POOL_ID='<identity pool id here>' COGNITO_USER_POOL_WEB_CLIENT_ID='<web client id here>' ./dev test browser -- open --config 'baseUrl=<insert your branch deployment url here>' 
```

Comment:
How do we get all this config when running `./dev local web --hybrid`? Maybe we could have a `./dev test browser --review-app` command?

Reply:
This was something I thought about and had some questions. Let's talk about this at our next 1:1?